nodecontrol/kmdControl_non_windows.go

```diff
@@ -0,0 +1,29 @@
+// +build !windows
```

Comment:
call the file `_unix.go` or `_common.go` as other non-windows files in the repo

Reply:
Done. Also moved the `+build` option below the copyright notice to match the location of other files